src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+const body = document.querySelector('body');
+
+const createNotification = (text, cssClass = '') =>
```

Comment:
Code quality issue: It's better to create an actual DOM element instead of a string. This way, you can manipulate the element more easily later. You can create an element using `document.createElement()`, and then set its text and class.